src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -437,9 +449,51 @@ private boolean worldTickSlow()
         lastOnlineTime = currTime;
 
         updateChildTime();
+        checkForUnload();
         return false;
     }
 
+    /**
+     * Check if we can unload the colony now.
```

Comment:
I'd call this updateChunkLoadTimer() and mention that its updating the timer or unloading the chunks when the time runs out